radio/src/storage/conversions/yaml/yaml_datastructs_commando8.cpp

```diff
@@ -0,0 +1,862 @@
+// generated by generate_yaml.py
+
+//
+// Enums first
+//
+
+const struct YamlIdStr enum_BacklightMode[] = {
```

Comment:
Please remove all changes in `storage/conversions/*`: you would only need them to support a pre-existing binary storage. As EdgeTx does not support binary storage for new radios, it should not be needed.